base/applications/calc/lang/de-DE.rc

```diff
@@ -307,89 +307,89 @@ END
 
 IDR_MENU_SCIENTIFIC_1 MENU
 BEGIN
-    POPUP "Bearbeiten"
+    POPUP "&Bearbeiten"
     BEGIN
-        MENUITEM "Kopieren\tSTRG-C", IDM_EDIT_COPY
-        MENUITEM "Einfügen\tSTRG-V", IDM_EDIT_PASTE
+        MENUITEM "&Kopieren\tStrg-C", IDM_EDIT_COPY
+        MENUITEM "&Einfügen\tStrg-V", IDM_EDIT_PASTE
     END
-    POPUP "Ansicht"
+    POPUP "&Ansicht"
     BEGIN
-        MENUITEM "Standard", IDM_VIEW_STANDARD
-        MENUITEM "Wissenschaftlich", IDM_VIEW_SCIENTIFIC
-        MENUITEM "Konversion", IDM_VIEW_CONVERSION
+        MENUITEM "&Standard", IDM_VIEW_STANDARD
+        MENUITEM "&Wissenschaftlich", IDM_VIEW_SCIENTIFIC
+        MENUITEM "Kon&version", IDM_VIEW_CONVERSION
         MENUITEM SEPARATOR
-        MENUITEM "Hex\tF5", IDM_VIEW_HEX, CHECKED
-        MENUITEM "Dezimal\tF6", IDM_VIEW_DEC, CHECKED
-        MENUITEM "Oktal\tF7", IDM_VIEW_OCT, CHECKED
-        MENUITEM "Binär\tF8", IDM_VIEW_BIN, CHECKED
+        MENUITEM "&Hex\tF5", IDM_VIEW_HEX, CHECKED
+        MENUITEM "De&zimal\tF6", IDM_VIEW_DEC, CHECKED
+        MENUITEM "O&ktal\tF7", IDM_VIEW_OCT, CHECKED
+        MENUITEM "&Binär\tF8", IDM_VIEW_BIN, CHECKED
         MENUITEM SEPARATOR
-        MENUITEM "Degree\tF2", IDM_VIEW_DEG, CHECKED
-        MENUITEM "Radiant\tF3", IDM_VIEW_RAD, CHECKED
-        MENUITEM "Grad\tF4", IDM_VIEW_GRAD, CHECKED
+        MENUITEM "&Deg\tF2", IDM_VIEW_DEG, CHECKED
+        MENUITEM "&Rad\tF3", IDM_VIEW_RAD, CHECKED
+        MENUITEM "&Grad\tF4", IDM_VIEW_GRAD, CHECKED
         MENUITEM SEPARATOR
-        MENUITEM "Zifferngruppierung", IDM_VIEW_GROUP, CHECKED
+        MENUITEM "Zifferngr&uppierung", IDM_VIEW_GROUP, CHECKED
     END
-    POPUP "Hilfe"
+    POPUP "&?"
```

Comment:
Why the change?

Reply:
Because that is what MS Calc uses here (pic from german MS XPSP3 calc attached)
![image](https://github.com/user-attachments/assets/449e12f3-f07a-412c-bf52-7a2670c5711a)